linux/net/rina/ipcp-instances.h

```diff
@@ -94,9 +94,19 @@ struct efcp_config {
         /* The data transfer constants */
         struct dt_cons * dt_cons;
 
+        /* Left here for phase 2 */
         struct policy * unknown_flow;
 };
 
+/* Represents the configuration of the RMT */
+struct rmt_config {
+	/* The PS name for the PDU Forwarding Table (Next hop) */
+	struct policy * pft_policy_set;
+
+	/* The PS name for the RMT */
+	struct policy * rmt_policy_set;
+};
+
 /* Represents a DIF configuration (policies, parameters, etc) */
```

Comment:
It is required in order to parse the netlink message coming from user space. Nevertheless I agree there is no need to keep it after setting the policy sets. I will fix that destroying this struct (which is created in the netlink layer) after setting the PS. Doing so, there is no need also to keep it in the rmt struct so I will remove that reference too.